datastream-kafka-connector/src/main/java/com/linkedin/datastream/connectors/kafka/KafkaConnectorUtils.java

```diff
@@ -0,0 +1,66 @@
+/**
+ *  Copyright 2020 LinkedIn Corporation. All rights reserved.
+ *  Licensed under the BSD 2-Clause License. See the LICENSE file in the project root for license information.
+ *  See the NOTICE file in the project root for additional information regarding copyright ownership.
+ */
+
+package com.linkedin.datastream.connectors.kafka;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import org.apache.commons.lang3.StringUtils;
+import org.codehaus.jackson.type.TypeReference;
+import org.slf4j.Logger;
+
+import com.linkedin.datastream.common.JsonUtils;
+import org.slf4j.LoggerFactory;
+
+
+/**
+ * Utility class for Kafka based connectors
+ */
+public class KafkaConnectorUtils {
+  private static final Logger LOG = LoggerFactory.getLogger(KafkaConnectorUtils.class.getName());
+  /**
+   * Reduce/Merge the KafkaTopicPartitionStatsResponse responses of a collection of host/instance into one response
+   */
+  public static String reduceTopicPartitionStatsResponses(Map<String, String> responses) {
```

Comment:
You might want to consider passing a logger to this method, or to a `KafkaConnectorUtils ` ctor. This way, log statements printed by this method would have the appropriate connector/task type name instead of `KafkaConnectorUtils`.

Reply:
Good point. Done.